docs/source/docs/index.rst

```diff
@@ -143,6 +143,7 @@ Read the research:
    ../examples/auto-failover
    ../reference/kubernetes/index
    ../running-jobs/distributed-jobs
```

Comment:
Shall we call this "Distributed Multinode Jobs"?

Reply:
Good point! Renamed. Thanks!